operations/boot_orchestration/Redeploy_the_IPXE_and_TFTP_Services.md

```diff
@@ -214,13 +216,13 @@ This procedure requires administrative privileges.
 
```

Comment:
Probably change the line above this one to say "Log into each iPXE pod"

Reply:
Pushed up a commit with this change.